metaworld/envs/mujoco/sawyer_xyz/sawyer_door_close.py

```diff
@@ -17,6 +17,8 @@ def __init__(self, random_init=False):
         self.obj_init_angle = self.init_config['obj_init_angle']
```

Comment:
can you rename this file/class (and other environment mods) SawyerDoorCloseEnvV2? This will allow us to merge it in master without modifying the v1 environments